pull-requests.md

```diff
@@ -48,13 +49,26 @@ If you need to step away for any reason, make a comment on the pull request or t
 If you haven't submitted a pull request before, you'll want to install WP-CLI for local development:
 
 1. Clone the WP-CLI git repository to your local machine: `git clone git@github.com:wp-cli/wp-cli.git ~/wp-cli`
-2. Install [Composer](https://getcomposer.org/) if you don't already have it.
-3. Run `composer install --prefer-source` to fetch all the dependencies.
-4. Run `./bin/wp --info` to test if everything was installed properly.
-
-Commands bundled with WP-CLI (e.g. `wp scaffold plugin`) will be editable from the `vendor/wp-cli` directory (e.g. `vendor/wp-cli/scaffold-command`). You'll need to fork the repository appropriately in order to have an `origin` to push to.
-
-Commands available for standalone installation (e.g. `wp dist-archive`) can be installed from source (e.g. `wp package install git@github.com:wp-cli/dist-archive-command.git`). Run `wp package path <package-name>` to find the appropriate directory to edit.
+2. Install [Composer](https://getcomposer.org/) and [hub](https://hub.github.com/) if you don't already have it.
```

Comment:
I'm wondering should there be a 3. `cd ~/wp-cli; hub fork` here? (Thanks for `hub` btw - new to me.)